src/lib/InstantSearch.js

```diff
@@ -365,8 +365,8 @@ See: https://www.algolia.com/doc/guides/building-search-ui/widgets/create-your-o
           this._isSearchStalled = true;
           this._render(
             this.helper,
-            this.helper.lastResults,
-            this.helper.lastResults._state
+            this.helper && this.helper.lastResults,
```

Comment:
I think this line here isn't needed, helper is always  going to be defined since we call methods on it